File: app/src/main/java/com/demo/demoapplication/ui/fragments/search/SearchViewModel.java
package com.demo.demoapplication.ui.fragments.search;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class SearchViewModel extends ViewModel {

    private final MutableLiveData<String> mText;

    public SearchViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("Hello world Search");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: app/src/main/java/com/demo/demoapplication/ui/fragments/search/SearchViewModel.kt
package com.demo.demoapplication.ui.fragments.search

import androidx.lifecycle.ViewModel
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.LiveData

class SearchViewModel : ViewModel() {
    private val mText: MutableLiveData<String> = MutableLiveData()
    val text: LiveData<String>
        get() = mText

    init {
        mText.value = "Hello world Search"
    }
}